Q3E/src/main/jni/doom3/neo/quake4/ai/AI_Move.cpp

```diff
@@ -3889,10 +3889,10 @@ class rvPathFinder {
 
 	/*
 	=====================
-	Success
+	success
```

Comment:
That and other changes with Success -> success is it really need? Looks like a mistake refactor in the file. Other bool functions stayed as it

Reply:
I see, that is X11 fix

Reply:
I removed include path `externlibs/include`, this headers only used in MinGW or MSVC, so don't need in Linux. so don't change function name of `Success`.